control/control.rnc

```diff
@@ -258,11 +257,18 @@ full_system_media_name =		element full_system_media_name { text }
 full_system_download_url =		element full_system_download_url { text }
 ## Defines if product needs precise time. It is short time workaround for bsc#1145193
 precise_time =		        	element precise_time { BOOLEAN }
-## SELinux mode: allows to choose the SELinux mode to be proposed (related to jsc#SLE-17427)
-selinux_mode = element selinux_mode { "disabled" | "permissive" | "enforcing" }
-## SELinux configurable: whether SELinux can be proposed/configured during installation
-## (related to jsc#SLE-17427)
-selinux_configurable = element selinux_configurable { BOOLEAN }
+## SELinux options
+selinux = element selinux {
+  MAP,
+    (
+      ## SELinux mode
+      element mode { "disabled" | "permissive" | "enforcing" }? &
+      ## Whether SELinux can be proposed/configured during installation
+      element configurable { BOOLEAN }? &
+      ## Required/Suggested patterns when using SELinux
```

Comment:
what is separated if more patterns are needed?

Reply:
I used the approach followed by `default_patterns`. I.e., a space separated list, already documented as you suggested.

Maybe it would have been better to have something like 

```xml
<patterns>
  <pattern>one-pattern</pattern>
  <pattern>another-pattern</pattern>
</patterns>
```

but I wanted to be not so disruptive with the other similar variables.